webapp/components/Map.tsx

```
@@ -1,27 +1,235 @@
-  useEffect(() => {
-    if (map.current || !mapContainer.current) return;
+interface MapProps {
+  pollutionData: Record<string, Prelevement>;
+}
 
-    map.current = new maplibregl.Map({
-      container: mapContainer.current,
-      style: "https://tiles.openfreemap.org/styles/positron",
-      center: [2.213749, 46.227638],
-      zoom: 5,
+export default function Map({ pollutionData }: MapProps) {
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)